wgsl/index.bs

```diff
@@ -176,7 +176,134 @@ Note: literals are parsed greedy. This means that for statements like `a -5`
 
 An identifier must not have the same spelling as a keyword or as a reserved keyword.
 
-## Attributes TODO ## {#attributes}
+## Attributes ## {#attributes}
+
+An <dfn noexport>attribute</dfn> modifies an object or type.
```

Comment:
This makes me realize we haven't defined "object".

I filed https://github.com/gpuweb/gpuweb/issues/1538 to sort that out.